Feature/custom calender/ap 2395

nolantellis opened the pull request.

Commit:
minor chnages

Commit:
Added tests and load test

Commit:
Merge remote-tracking branch 'origin/development' into feature/custom-calender/AP-2395

raboczi committed:
Added license headers to Apromore-Calendar, and added it as a module of the top-level pom.xml

raboczi reviewed (changes requested):
I added some auto-generated license headers and added Apromore-Calendar to the top-level pom.xml.

Could you rename CalenderNotExistsException to org.apromore.calendar.exception.CalendarNotExistsException?

Commit:
Correction of code due to rebase of development

nolantellis commented:
> I added some auto-generated license headers and added Apromore-Calendar to the top-level pom.xml.
> 
> Could you rename CalenderNotExistsException to org.apromore.calendar.exception.CalendarNotExistsException?

Done
